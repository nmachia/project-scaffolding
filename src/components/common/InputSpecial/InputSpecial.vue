<template>
	<div class="input-special">
		<input type="text" class="input-special__field" aria-label="some input" />
	</div>
</template>

<script>
	export default {
		name: 'InputSpecial',
		data() {
			return {};
		},
	};
</script>
