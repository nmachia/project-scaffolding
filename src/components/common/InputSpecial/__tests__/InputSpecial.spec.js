import { mount, createLocalVue } from '@vue/test-utils';
import InputSpecial from '../InputSpecial.vue';

const localVue = createLocalVue();

describe('InputSpecial', () => {
	let wrapper;
	beforeEach(() => {
		wrapper = mount(InputSpecial, {
			localVue,
		});
	});

	describe('General Behavior', () => {
		it('renders correctly', () => {
			expect(wrapper.element).toMatchSnapshot();
		});
	});
});
