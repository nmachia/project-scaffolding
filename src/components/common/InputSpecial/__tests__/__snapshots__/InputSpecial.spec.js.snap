// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputSpecial General Behavior renders correctly 1`] = `
<div
  class="input-special"
>
  <input
    aria-label="some input"
    class="input-special__field"
    type="text"
  />
</div>
`;
